docs/zz_generated.kubermaticConfiguration.ce.yaml

```diff
@@ -381,6 +381,9 @@ spec:
       dockerTagSuffix: ""
     # APIServerReplicas configures the replica count for the API-Server deployment inside user clusters.
     apiserverReplicas: 2
+    # DefaultApplications contains configuration for default Application settings. This is an ee-only feature.
+    defaultApplications:
+      namespace: ""
```

Comment:
We should rather keep the encapsulating struct generic and call it `applications` or `applicationSettings`. Since default applications != enforced applications and I don't think we should in the future add a separate block for `enforcedApplications` or just `applications`(for generic apps) if we want to expose additional configurations.